### DISCUSSION TITLE
Organizing and linking of GitHub and Jupyter

### DISCUSSION THREAD

**Discussion by laurel-nave-powers** [sr320/course-fish546-2021#55] [Q&A] [answered]
I am confused about how to organize and link my Github repository with my Jupyter-lab. I watched the class recording from last class and tried to follow along but it doesn't seem to be working for me. I cloned my repo to my GitHub desktop. I found where my repo folder is on my computer and moved it to my desktop in a folder called GitHub. But when I go into Jupyter-lab there is no sign of any of those folders. 
Maybe we could go over this in class today? 
Thanks!

**Comment by sr320** [comment]
One problem is that after you clone the repo from GitHub to your computer, you moved the directory. That breaks everything. 
But that should not impact your ability to see folders that are present on your computer via Jupyter lab.
Yes we should have time to go through this in class.

**Comment by kubu4** [comment, marked as answer]
From today's class here's what was addressed:

> I found where my repo folder is on my computer and moved it to my desktop in a folder called GitHub.

Deleted existing `GitHub/` directory, removed from GitHub Desktop app, then re-cloned directly to `~/Desktop/GitHub`. Discussed _not_ moving folder after cloning.

> when I go into Jupyter-lab there is no sign of any of those folders.

Primary issue was launching Jupyter Lab from the installation location. Since Jupyter Lab can only navigate _down_ directories, there is/was no way to browse to the desired folder.

So, solution was to use Terminal to navigate to `~/Desktop/GitHub/` first. Then, launch Jupyter Lab via Terminal with just the command `jupyter-lab`
